www/src/components/users/MagicLogin.tsx

```diff
@@ -348,12 +208,11 @@ function OAuthOptions({ oauthUrls }: any) {
             wrap="wrap"
             flexWrap="wrap"
           >
-            {oauthUrls.map(url => (
+            {[...oauthUrls].sort(sortOauthUrls).map(url => (
```

Comment:
why do you need to create the newlist here?

Reply:
I didn't write that, but it does prevents sort() from mutation oauthUrls, which is not strictly necessary in this case, but I think keeping things immutable is generally good practice.